src/components/departure_list/list.js

```diff
@@ -1,32 +1,33 @@
 // @flow
 
 import React from 'react';
-import { Button, SectionList, View } from 'react-native';
-
+import { Button, SectionList, View, Text } from 'react-native';
+import { observer } from 'mobx-react';
 import Departure from './departure';
 import Header from './header';
 import styles from '../../styles';
+import { store } from '../../store/schedule.js';
 
 
 // NOTE: this is temporary data until MobX is hooked up
-const SECTIONS = [
-    {
-        data: [
-            { key: 'something', time: '12:34', name: 'Union 12:34 - Georgetown 13:33' },
-            { key: 'somethign', time: '12:34', name: 'Union 12:34 - Georgetown 13:33' },
-        ],
-        routeColor: '#0000BB',
-        title: 'Kitchener',
-    },
-    {
-        data: [
-            { key: 'somethgin', time: '12:34', name: 'Union 12:34 - Milton 13:33' },
-            { key: 'sometghin', time: '12:34', name: 'Union 12:34 - Milton 13:33' },
-        ],
-        routeColor: '#FF0000',
-        title: 'Milton',
-    },
-];
+// const SECTIONS = [
+//     {
+//         data: [
+//             { key: 'something', time: '12:34', name: 'Union 12:34 - Georgetown 13:33' },
+//             { key: 'somethign', time: '12:34', name: 'Union 12:34 - Georgetown 13:33' },
+//         ],
+//         routeColor: '#0000BB',
+//         title: 'Kitchener',
+//     },
+//     {
+//         data: [
+//             { key: 'somethgin', time: '12:34', name: 'Union 12:34 - Milton 13:33' },
+//             { key: 'sometghin', time: '12:34', name: 'Union 12:34 - Milton 13:33' },
+//         ],
+//         routeColor: '#FF0000',
+//         title: 'Milton',
+//     },
+// ];
```

Comment:
Why is this just commented out? Don't we usually remove unused code?

Reply:
Yeah, will remove